base/sysinfo.jl

```diff
@@ -16,28 +16,55 @@ export  CPU_CORES,
         free_memory,
         total_memory
 
-import ..Base: WORD_SIZE, OS_NAME, ARCH, MACHINE
-import ..Base: show, uv_error
+import ..Base: show
 
 global CPU_CORES
+"""
+    Sys.CPU_CORES
+
+   The number of CPU cores in the system.
+"""
+:CPU_CORES
+
+"""
+    Sys.ARCH
+
+A symbol representing the architecture of the build configuration.
+"""
+const ARCH = ccall(:jl_get_ARCH, Any, ())
+
+
+"""
+    Sys.KERNEL
+
+A symbol representing the name of the operating system, as returned by uname of the build configuration.
```

Comment:
I still don't think we should have this variable, but if we do it should have backticks around `uname` in the docstring